src/chi/components/steps/steps.scss

```diff
@@ -0,0 +1,91 @@
+@import 'variables';
+@import 'mixins';
+
+.chi {
+
+  .a-steps {
+    display: table;
+    padding: 0;
+    table-layout: fixed;
+    width: 100%;
+
+    & > li {
+      color: $brand-color;
+      display: table-cell;
+      font-size: $text;
+      line-height: 1rem;
+      position: relative;
+      text-align: center;
+      z-index: 0;
+
+      &::before {
+        background-color: $brand-color;
+        border: 0.125rem solid $text-white;
+        border-radius: 50%;
+        content: '';
+        display: block;
+        height: 1rem;
+        line-height: 1rem;
+        margin: 0 auto 0.5rem;
+        width: 1rem;
+      }
+
+      &::after {
+        background-color: $brand-color;
+        content: '';
+        height: 0.125rem;
+        left: 50%;
+        position: absolute;
+        top: 0.5rem;
```

Comment:
@davidojedacl @jllr to fix the vertical alignment of the circle, we need to adjust the `::after`, not the `::before`. the top value here should be `top: .4375rem;` and that should do it.